src/api.cpp

```diff
@@ -726,6 +726,12 @@ cl_int CLVK_API_CALL clGetDeviceInfo(cl_device_id dev,
         size_ret = sizeof(val_uint);
         break;
     case CL_DEVICE_MAX_READ_WRITE_IMAGE_ARGS:
+        if (!device->supports_read_write_images()) {
+            val_uint = 0;
+            copy_ptr = &val_uint;
+            size_ret = sizeof(val_uint);
```

Comment:
```suggestion
            size_ret = sizeof(val_uint);
            break;
```

Is this missing a `break`? Unless I'm missing something, `val_uint` will always be overwritten below.